starters/angular/basic/src/app/pages/home/home.component.html

```diff
@@ -0,0 +1,42 @@
+<h1 class="large-heading">Hello, My App</h1>
+
+<section class="features">
+  <article class="card">
+    <h2>Server Side Rendering (SSR)</h2>
+    <p>Dynamic content is generated by the server running on Cloud Functions. Improve performance, SEO and achieve better Core Web Vitals results.
+      Visit <a href="ssr">/ssr</a> to see the Angular SSR in action.</p>
+  </article>
+  <article class="card">
+    <h2>Prerendering (SSG)</h2>
+    <p>Prerendering maintains the same performance benefits of server-side rendering (SSR) but achieves a reduced Time to First Byte (TTFB),
+      ultimately enhancing user experience. The key distinction lies in its approach that pages are served as static content, and there is no request-based rendering.
+      Visit <a href="ssg">/ssg</a> page to see a statically generated page served by Firebase Hosting.</p>
+  </article>
+  <article class="card">
+    <h2>Deferrable Views</h2>
+    <p>
+      Take advantage of one of the most important features of Angular from recent releases.
+      Deferrable views are a powerful tool that can be used to reduce the initial bundle size of your application or defer heavy components that may not ever be loaded until a later time.
+      Visit <a routerLink="deferrable-views">/defer</a> page to see defer in Action.
+    </p>
+  </article>
+</section>
+
+<section class="links-container">
```

Comment:
@jhuleatt @jamesdaniels please let me know what links should be displayed here.

Reply:
FWIW the links were removed in the Next.js boilerplate

Reply:
let's remove the links, but keep the CSS please so we can easily add them back in later.